tests/integration/test_disks_app_func/test.py

```diff
@@ -0,0 +1,375 @@
+        "ENGINE=MergeTree() "
+        "ORDER BY word SETTINGS storage_policy = 'test1'"
+    )
+
+    source.query("INSERT INTO test_table(*) VALUES ('test1', 2)")
+
+    docker_api = started_cluster.docker_client.api
+
+    cmd = [
+        "/usr/bin/clickhouse",
+        "disks",
+        "copy",
+        "--diskFrom",
+        "test1",
+        "--diskTo",
+        "test2",
+    ]
+
+    instance = started_cluster.instances["disks_test_cp"]
+    container = instance.get_docker_handle()
+
+    exec_id = docker_api.exec_create(container.id, cmd, stderr=True)
+    docker_api.exec_start(exec_id)
+
+    cmd = [
+        "/usr/bin/clickhouse",
+        "disks",
+        "--disk",
+        "test2",
+        "list",
+    ]
+
+    exec_id = docker_api.exec_create(container.id, cmd, stderr=True)
+    output = docker_api.exec_start(exec_id).decode("utf8")
+    print(output)
+    files = output.split("\n")
+
+    pos = 0
+    while re.search("configuration file", files[pos]):
+        pos = pos + 1
+
+    assert files[pos] == "path1"
+
+
+def test_disks_app_func_ln(started_cluster):
+    source = cluster.instances["disks_test_ln"]
+
+    source.query("DROP TABLE IF EXISTS test_table")
+
+    source.query(
+        "CREATE TABLE test_table(word String, value UInt64) "
+        "ENGINE=MergeTree() "
+        "ORDER BY word SETTINGS storage_policy = 'test1'"
+    )
+
+    source.query("INSERT INTO test_table(*) VALUES ('test1', 2)")
+
+    docker_api = started_cluster.docker_client.api
+
+    cmd = [
+        "/usr/bin/clickhouse",
+        "disks",
+        "link",
+        "--diskTo",
+        "test2",
+        "data/default/test_table",
+        "tester",
+    ]
+
+    instance = started_cluster.instances["disks_test_ln"]
+    container = instance.get_docker_handle()
+
+    exec_id = docker_api.exec_create(container.id, cmd, stderr=True)
+    docker_api.exec_start(exec_id)
+
+    cmd = [
+        "/usr/bin/clickhouse",
+        "disks",
+        "--disk",
+        "test2",
+        "list",
+    ]
+
+    exec_id = docker_api.exec_create(container.id, cmd, stderr=True)
+    output = docker_api.exec_start(exec_id).decode("utf8")
+    print(output)
+    files = output.split("\n")
+
+    pos = 0
+    while re.search("configuration file", files[pos]):
+        pos = pos + 1
+
+    assert files[pos] == "tester"
+
+
+def test_disks_app_func_rm(started_cluster):
+    source = cluster.instances["disks_test_rm"]
+
+    source.query("DROP TABLE IF EXISTS test_table")
+
+    source.query(
+        "CREATE TABLE test_table(word String, value UInt64) "
+        "ENGINE=MergeTree() "
+        "ORDER BY word SETTINGS storage_policy = 'test1'"
+    )
+
+    source.query("INSERT INTO test_table(*) VALUES ('test1', 2)")
+
+    docker_api = started_cluster.docker_client.api
+
+    cmd = [
+        "/usr/bin/clickhouse",
+        "disks",
+        "copy",
+        "--diskFrom",
+        "test1",
+        "--diskTo",
+        "test2",
+    ]
```

Comment:
Why not make it the same as in shell? like `/usr/bin/clickhouse disks cp <old_path> <new_path>` ? This is quicker and more familiar.